### PR TITLE
Add django girls pycon post

### PR DESCRIPTION
**Pull request by RachellCalhoun**
Adding info about dggr participation at pycon

### CODE REVIEW

File: content/post/2018-djangogirls-pycon.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+lastmod: 2018-05-30
+draft: false
+tags: ["events", "2018", "djangogirls", "pycon"]
+categories: ["djangogirls", "2018"]
+---
+![2018 Sponsors for Django Girls Workshop](https://pbs.twimg.com/media/Dc3BXYiXkAE1fKY.jpg)
+
+## Django Girls GR at PyCon
+
+Django Girls Grand Rapids was honored to be asked to manage the Django Girls booth at Pycon in Cleveland, Ohio this year. Because of our participation, we were able to sponsor two volunteers' tickets to PyCon that might not otherwise been able to go.
```
Comment (jacebrowning) [inline content/post/2018-djangogirls-pycon.md]: Pycon => PyCon

File: content/post/2018-djangogirls-pycon.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+
+There were three people total that volunteered to help out with the booth. They spent days up to the event preparing, and 3 days at the booth answering questions, inspiring new organizers, and showing PyCon how to have a fun booth.
+
+We had a blast, PyCon, can't wait to see you next time!
+
+## Django Girls Rocking Booth
```
Comment (jacebrowning) [inline content/post/2018-djangogirls-pycon.md]: Should this be `Girls'`?

File: content/post/2018-djangogirls-pycon.md [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+
+Although our organization is run completely on donations and sponsors, we were able to put together a fun booth that got a lot of the attendees attention.
```
Comment (jacebrowning) [inline content/post/2018-djangogirls-pycon.md]: `attendees'`

**Review by jacebrowning — CHANGES_REQUESTED** [review]

**Comment by RachellCalhoun** [comment]
@jacebrowning thanks for catching those! I corrected them. lmk if there is anything else

**Review by jacebrowning — APPROVED** [review]

**Pull request merged**